src/PandoraMonitoring.cc

```diff
@@ -74,6 +78,9 @@
 #include <unordered_map>
 #include <vector>
 
+#include <TEveProjectionManager.h>
+#include <TEveProjectionAxes.h>
```

Comment:
Probably ought to replace angle brackets with "" (angle brackets conventionally for standard or system includes).

Also move into list of ROOT includes.

Reply:
Actually I missed these when checking the code and they can be removed, which I will do now.